### ISSUE TITLE
Bithub doesn't seem to be activated in signal desktop

### ISSUE THREAD

**Issue by ta1337** [WhisperSystems/BitHub#58]
@ikarulus made an amazing dark theme for the chrome extension. It was merged but the contrbuter still didn't get his reward, although the contributer did not marked this as freebe.
I'm not really in OWS so maybe this is not activated in desktop by default, but I thought, maybe it's a good start for my first issue here.
Sorry for my bad english.
PR: https://github.com/WhisperSystems/Signal-Desktop/pull/974
Commit: https://github.com/WhisperSystems/Signal-Desktop/commit/53cd3af78b220e29be04bf613e2d8ec6ef155f12

**Comment by ikarulus** [comment]
This is right, I did not get the reward. But when I submitted the PR, I had github's email privacy option enabled. This isn't probably the reason for the problem as github states below the option:"We will use ikarulus@users.noreply.github.com when performing web-based Git operations and sending email on your behalf. If you want command line Git operations to use your private email you must set your email in Git.".
I was not commiting via github's web-based interface and if you check the repository's log, you will find my email address. I don't think Bithub check whether email privacy is enabled. Why should it when the address is in the logs anyway?

**Comment by ta1337** [comment]
Maybe we ask @moxie0? He is also the maintainer of BitHub, am i right?

**Comment by Jarwain** [comment]
http://bithub.whispersystems.org/

Based on the dashboard it seems that bithub not being enabled on Signal Desktop is intentional. It also seems like bithub isn't being actively developed. If you really want to contact a maintainer I'd try the mailing list thats in the Readme.md or, failing that, Twitter.

**Comment by ta1337** [comment]
https://github.com/WhisperSystems/Signal-Desktop/issues/1006https://github.com/WhisperSystems/Signal-Desktop/issues/1006

**Comment by ikarulus** [comment]
I think this is a BitHub issue as well.
